ignite/pkg/gomodule/gomodule.go

```diff
@@ -166,7 +166,17 @@ func JoinPath(path, version string) string {
 
 // FindModule returns the Go module info for an import path.
 // The module is searched within the dependencies of the module defined in root dir.
+// If a local module path is passed, it returns the local module info.
 func FindModule(ctx context.Context, rootDir, path string) (Module, error) {
+	// can be a local module.
+	if filepath.IsAbs(path) || strings.HasPrefix(path, ".") { // indicates that this is a local module.
+		return Module{
+			Path:    path,
+			Version: "",
```

Comment:
I'm wondering if we should resolve the version maybe by checking the last local module tag, making git mandatory. Not having the version might bite us at some point. @julienrbrt do you maybe have an opinion or thought about it?

Reply:
From what I've seen version isn't used anywhere, so that's why I left it empty. We only check the version of the sdk iirc.
Verifying if git exists could work, and we could keep it optional, however we would need to take the latest commit and build the correct go pseudo version. Not sure what would be the benefit tbh, but that's something I can investigate if wanted.